gas/config/tc-arc.c

```diff
@@ -1887,6 +1890,7 @@ find_opcode_match (const struct arc_opcode_hash_entry *entry,
 		    case O_gotoff:
 		    case O_gotpc:
 		    case O_pcl:
+		    case O_jli:
```

Comment:
Here we need to move O_jli in a stand alone case, as all other relocs are require for LIMM. So:
case O_jli:
if (!(operand->flags & ARC_OPERAND_SIGNED))
  goto match_failed;
break;